README.md

```diff
@@ -998,16 +1182,26 @@ RFC: [Remove deprecated functionality in PHP 7](https://wiki.php.net/rfc/remove_
 
 ### Reclassification and Removal of E_STRICT Notices
 
-E_STRICT notices have always been a bit of a grey area in their meaning. This changes removes this error category altogether and either: removes the E_STRICT notice, changes it to an E_DEPRECATED if the functionality will be removed in future, changes it to an E_NOTICE, or promotes it to an E_WARNING.
+E_STRICT notices have always been a bit of a grey area in their meaning. This
+changes removes this error category altogether and either: removes the E_STRICT
+notice, changes it to an E_DEPRECATED if the functionality will be removed in
+future, changes it to an E_NOTICE, or promotes it to an E_WARNING.
 
 **BC BREAKS**
- - Because E_STRICT is in the lowest severity error category, any error promotions to an E_WARNING may break custom error handlers.
+ - Because E_STRICT is in the lowest severity error category, any error
+   promotions to an E_WARNING may break custom error handlers.
 
 RFC: [Reclassify E_STRICT notices](https://wiki.php.net/rfc/reclassify_e_strict)
 
 ### Deprecation of Salt Option for `password_hash()`
 
-With the introduction of the new password hashing API in PHP 5.5, many began implementing it and generating their own salts. Unfortunately, many of these salts were generated from cryptographically insecure functions like mt_rand(), making the salt far weaker than what would have been generated by default. (Yes, a salt is always when hashing passwords with this new API!) The option to generate salts have therefore been deprecated to prevent developers from creating insecure salts.
+With the introduction of the new password hashing API in PHP 5.5, many began
+implementing it and generating their own salts. Unfortunately, many of these
+salts were generated from cryptographically insecure functions like mt_rand(),
+making the salt far weaker than what would have been generated by default.
+(Yes, a salt is always when hashing passwords with this new API!) The option to
```

Comment:
Missing a word, it should be "...a salt is always **used** when hashing passwords..."